apps/sptribs/sptribs-case-api/aat.yaml

```diff
@@ -6,3 +6,5 @@ spec:
   values:
     java:
       environment:
+        GLOBAL_SEARCH_MIGRATION_ENABLED: true
+        GLOBAL_SEARCH_MIGRATION_TEST_CASE_REF: 1709541736486585
```

Comment:
This one needs to be a string -> "1709541736486585" otherwise there will be an error parsing the Long

Reply:
Ok sure